avatar2/targets/angr_target.py

```diff
@@ -69,7 +71,7 @@ def fill_page_from_remote(self, state):
             start, value = self._read_memory(state, x,
                                              x+state.arch.bytes)[0]
             value._object = value.object.reversed
-            value.object = value.object.reversed
+            #value.object = value.object.reversed
```

Comment:
This line was not present in the old avatar version (for which I developed my patch initially), it was added in commit 65808cf, but it seems to break normal working - maybe it's a bug? I commented it out

Reply:
I think this was to deal with some endianness issues - which of course behaves differently if your target is BE. :) Ideally, we would advance the testcase to also compile to little-endian MIPS and test it there as well